### PR TITLE
Init engine

### PR DESCRIPTION
**Pull request by kolyastepanets**

### CODE REVIEW

File: app/assets/javascripts/shareconomy/application.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// This is a manifest file that'll be compiled into application.js, which will include all the files
```
Comment (sparrow) [inline app/assets/javascripts/shareconomy/application.js]: do we need application.js file for the current project?

File: app/assets/stylesheets/shareconomy/application.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/*
```
Comment (sparrow) [inline app/assets/stylesheets/shareconomy/application.css]: do we need application.css file for the current project?

File: app/views/layouts/shareconomy/application.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```
Comment (sparrow) [inline app/views/layouts/shareconomy/application.html.erb]: do we need layout for the project?

**Comment by kopchak** [comment]
ok

**Pull request merged**